financeager/rich.py

```diff
@@ -1,17 +1,23 @@
+from typing import TYPE_CHECKING, Any
+
 from rich import box
 from rich.panel import Panel
 from rich.table import Table
+from rich.table import Table as RichTable
 
 from . import DEFAULT_BASE_ENTRY_SORT_KEY, DEFAULT_CATEGORY_ENTRY_SORT_KEY
 
+if TYPE_CHECKING:
+    from .listing import Listing
+
 
 def richify_listings(
-    listings,
-    stacked_layout=False,
-    category_sort=None,
-    category_percentage=False,
-    entry_sort=None,
-):
+    listings: list["Listing"] | None,
```

Comment:
why do you put "Listing" as string here instead of using `Listing` class from the import?